src/status_im/contexts/wallet/swap/events.cljs

```diff
@@ -161,22 +190,46 @@
                                :swap-proposal          (when-not (empty? best-routes)
                                                          (assoc (first best-routes) :uuid request-uuid))
                                :error-response         error-response
-                               :loading-swap-proposal? false)}
+                               :loading-swap-proposal? false
+                               :initial-response?      false)}
+         (and initial-response? (seq best-routes))
+         (assoc :fx
+                [[:dispatch
+                  [:centralized-metrics/track :metric/swap-proposal-received
+                   {:receive_token  (:symbol asset-to-receive)
+                    :receive_amount (-> amount-out
```

Comment:
@briansztamfater @shivekkhurana I see in the PR we are tracking amounts. Analytics has a a high opt-out rate compared to opt-in, which means the aggregation of amounts will be quite imprecise. It's different for example than extrapolating button presses from the opt-out rate, here with amounts a single user that opted out can have more impact than all other combined users.

Shouldn't we keep track of swap amounts from external tools? I was assuming we can calculate revenue and swaps very precisely, so there would be no need to track amounts in MixPanel.

Is my assessment correct? In which case why would we track amounts in MixPanel?

Reply:
I don't have any specific preference, tbh. I think if we remove amounts we may just miss some metrics that can tell how users interact with the feature (i.e. swap proposal requests vs. swap proposal actually executed).

If there's no value on tracking amounts through MixPanel, I can remove them. I assume we can track revenue on provider's dashboards, but this may provide other extra (maybe) useful data.

Reply:
The other data seems fine to capture, seems useful, just the amounts seem unnecessary to me because we have better ways to report revenue (or if we don't we should have instead of depending on MixPanel with low opt-in rate).